### ISSUE TITLE
There seems to be an error in the HTML 5.1 specification outline example.

### ISSUE THREAD

**Issue by ghost** [w3c/html#956]
Hi, I am looking closely to understand the part of the outline algorithm.
Then, I found that some of the examples in the outline algorithm seem to have errors.

In Example 28 &mdash; [headings-and-sections](https://www.w3.org/TR/html51/sections.html#headings-and-sections)

```html
<body>
  <h1>Apples</h1>
  <p>Apples are fruit.</p>
  <section>
    <h2>Taste</h2>
    <p>They taste lovely.</p>
    <h3>Sweet</h3>
    <p>Red apples are sweeter than green ones.</p>
    <h3>Color</h3>
    <p>Apples come in various colors.</p>
  </section>
</body>
```

```html
<body>
  <h1>Apples</h1>
  <p>Apples are fruit.</p>
  <section>
    <h2>Taste</h2>
    <p>They taste lovely.</p>
    <section>
      <h3>Sweet</h3>
      <p>Red apples are sweeter than green ones.</p>
    </section>
  </section>
  <section>
    <h3>Color</h3>
    <p>Apples come in various colors.</p>
  </section>
</body>
```

It describes that 
> 'Both of the documents above are semantically identical and would produce the same outline in compliant user agents'.

But I think the two have different outlines as follows.
( And the "Nu Html Checker" showed the same result. )

*outline for first code fragment*
> + "Apple"
>     - "Taste"
>         - "Sweet"
>         - "Color"

* outline for second code fragment*
> + "Apple"
>     - "Taste"
>         - "Sweet"
>     - "Color"

What's wrong?

And I also found that the same example (second example) was modified in HTML 5.2 Specification as follows.

```html
<body>
  <h1>Apples</h1>
  <p>Apples are fruit.</p>
  <section>
    <h2>Taste</h2>
    <p>They taste lovely.</p>
    <section>
      <h3>Sweet</h3>
      <p>Red apples are sweeter than green ones.</p>
    </section>
    <section>
      <h3>Color</h3>
      <p>Apples come in various colors.</p>
    </section>
  </section>
</body>
```

If this is correct, I think the outline of the first example is exactly the same as the outline of this example.

I would like to know if the example in HTML 5.1 is wrong, if the example in HTML 5.2 is correct, or if I misunderstood the example in HTML 5.1.

**Comment by chaals** [comment]
@seongbong as I understand it you have correctly understood the algorithm, and the example in the 5.1 version was presented incorrectly, which is why it was changed.

ping @stevefaulkner @siusin

**Comment by mulder21c** [comment]
@chaals aha! I see. Thanks for your comment

**Comment by siusin** [comment]
Thanks @mulder21c !

We will fix this error in the Proposed REC of HTML 5.1 2nd Edition, which will be published in the end of this month.

Also see https://rawgit.com/w3c/html/html5.1-2/single-page.html#headings-and-sections.
